provision/vvv-init.sh

```diff
@@ -61,3 +61,22 @@ else
     sed -i "s#{{TLS_CERT}}##" "${VVV_PATH_TO_SITE}/provision/vvv-nginx.conf"
     sed -i "s#{{TLS_KEY}}##" "${VVV_PATH_TO_SITE}/provision/vvv-nginx.conf"
 fi
+
+DELETE_DEFAULT_PLUGINS=`get_config_value 'delete_default_plugins' ''`
+if [ ! -z "${DELETE_DEFAULT_PLUGINS}" ]; then
+    noroot wp plugin delete akismet
+    noroot wp plugin delete hello
+fi
+
+WP_PLUGINS=`get_config_value 'plugins' ''`
+if [ ! -z "${WP_PLUGINS}" ]; then
+    for plugin in ${WP_PLUGINS//- /$'\n'}; do 
+        noroot wp plugin install "${plugin}" --activate
+    done
+fi
+
+WP_LOCALE=`get_config_value 'locale' ''`
+if [ ! -z "${WP_LOCALE}" ]; then
+    noroot wp language core install "${WP_LOCALE}"
```

Comment:
It'd be good if we can test if the locale is already installed and avoid an additional network request, but does WP CLI already do this internally? There's an is installed subcommand here https://developer.wordpress.org/cli/commands/language/core/is-installed/

```sh
wp language core is-installed de_DE
```

Reply:
In my testing is already done by WP-CLI, if it is already installed will print like an error but the provision will move on.

Reply:
good to know, but lets check anyway, if only to keep provisioning output nicer